### ISSUE TITLE
Moderate wcag issues in altinn appfrontend

### ISSUE THREAD

**Issue by jeevananthank** [Altinn/app-frontend-react#28]
## Describe the bug
Altinn app frontend has moderate WCAG issues and the automated tests are failing.

## To Reproduce
1. Install the axe devtools chrome [plugin](https://chrome.google.com/webstore/detail/axe-devtools-web-accessib/lhdoppojpmngadmnindnejefpokejbdd)
2. Start an app instance and run the axe scan from the dev tools
3. Moderate issues are reported by the scan.

## Expected behavior
No issues should be reported.

## Screenshots
![image](https://user-images.githubusercontent.com/49391725/130199520-257f21e0-e59b-4d87-9521-0e720c8dfc1c.png)

## Additional info
env: at22
app: ttd/apps-test

**Comment by Febakke** [comment]
This should be fixed as a part of #10 
Closing this issue
